content/en/continuous_integration/pipelines/_index.md

```diff
@@ -24,7 +24,23 @@ cascade:
+| {{< ci-details title="Manual steps" >}}The ability to identify when there is a job with a manual approval phase in the overall pipeline, which is split into two pipelines: <br>- From the beginning of the pipeline to when the manual approval action starts and the pipeline becomes <code>Blocked</code>.<br>- From the end of the manual approval action to the end of the pipeline.<br>{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  | {{< X >}} |  |
+| {{< ci-details title="Queue time" >}}The ability to add the time for which a job (or pipeline depending on the CI provider) was in the queue before it executes.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |
+| {{< ci-details title="Logs correlation" >}}The ability to retrieve pipeline or job logs from the CI provider. Logs are displayed on the <strong>Logs</strong> tab in the Pipeline Execution view.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  |  | {{< X >}} |  |  |  |
+| {{< ci-details title="Infrastructure metric correlation" >}}The ability to display host-level information for the Datadog Agent, pipelines, or job runners.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  |  |  |
+| {{< ci-details title="Custom spans for traced commands using datadog-ci" >}}The ability to send command-level events to CI Visibility so the events can be incorporated in the pipeline flame graph. You can then query and analyze <a href="https://docs.datadoghq.com/continuous_integration/pipelines/custom_commands/">these events</a>. {{< /ci-details >}} | {{< X >}} |  | {{< X >}} |  | {{< X >}} |  |  |  |
+| {{< ci-details title="Custom predefined tags" >}}The ability to set static pipeline tags that do not change between executions in a CI provider.{{< /ci-details >}} | {{< X >}} | {{< X >}} | {{< X >}} |  | {{< X >}} | {{< X >}} |  |  |
+| {{< ci-details title="Custom tags and metrics at runtime" >}}The ability to add <a href="https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/">user-defined text and numerical tags</a> to pipelines and jobs in CI Visibility.{{< /ci-details >}} | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} |  | {{< X >}} |  |  |
+| {{< ci-details title="Parameters" >}}The ability to add the custom pipeline parameters that users set (for example, <code>DYNAMICS_IS_CHILD:true</code>). Users can search using these parameters in the Datadog UI to find all the events with a specific parameter.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  |  |  |  | {{< X >}} |  |
+| {{< ci-details title="Pipeline failure reason" >}}The ability to add the reason for a failure of a pipeline or a job.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  |  |  |  | {{< X >}} | {{< X >}} |
```

Comment:
Identification of specific reason behind pipeline or job failure

Reply:
```suggestion
| {{< ci-details title="Pipeline failure reason" >}}Identification of a specific reason behind a pipeline or job failure.{{< /ci-details >}} | {{< X >}} | {{< X >}} |  |  |  |  | {{< X >}} | {{< X >}} |
```